gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -5,6 +5,22 @@
         <li class="active"><a data-toggle="tab" href="#settings-general" translate>General</a></li>
         <li><a data-toggle="tab" href="#settings-gui" translate>GUI</a></li>
         <li><a data-toggle="tab" href="#settings-connections" translate>Connections</a></li>
+        <!-- Use the value from the real config here, so that tabs don't disappear upon items being removed -->
+        <li ng-if="config.remoteIgnoredDevices.length > 0">
```

Comment:
I don't like UI elements that disappear. Imagine a debug situation where we ask a user to check whether the device in question is listed in the the ignored devices tab - which they won't find then -> confusion. I'd prefer the tab to show with the number of device indication showing `0`. You even have that case already covered in the body of the tab: https://github.com/syncthing/syncthing/pull/5084/files#diff-b601b30b54aeaa77dad2c1fcb487aaadR249

Reply:
Good point.